requirements/dev.txt

```diff
@@ -11,7 +11,7 @@ pytest~=6.2.5
 parameterized~=0.7.4
 
 # Integration tests
-click~=7.1
+click~=8.0
```

Comment:
We could get rid of this TBH, it's used for terminal styling only.

Reply:
haven't checked but yeah